SEGGER/SEGGER_RTT.c

```diff
@@ -1892,6 +1893,28 @@ int SEGGER_RTT_SetFlagsDownBuffer(unsigned BufferIndex, unsigned Flags) {
   return r;
 }
 
+#if defined(CONFIG_SEGGER_RTT_INIT_MODE_STRONG_CHECK)

```

Comment:
Where is `CONFIG_SEGGER_RTT_INIT_MODE_STRONG_CHECK` declared?

Reply:
same as above